CMakeLists.txt

```diff
@@ -78,38 +125,16 @@ file(GLOB_RECURSE SRC_FILES
         )
 
 add_executable(${PROJECT_NAME} ${SRC_FILES})
-add_dependencies(${PROJECT_NAME} gatb gtest)
+
+add_dependencies(boost gatb gtest)
+add_dependencies(${PROJECT_NAME} gatb gtest boost)
```

Comment:
Is this repeated on purpose? (with the addition of the project name)

Reply:
yeah, we could remove `gatb` and `gtest` from the second one, but I left just for documentation purposes... if we don't specify this, we have compilation issues that I don't exactly remember which one